lineman/app/components/edit_group_form/edit_group_form.haml

```diff
@@ -3,76 +3,139 @@
-
-      .checkbox
-        %label{for: 'group-members-can-create-subgroups'}
-          %input.edit-group-form__members-can-create-subgroups#group-members-can-create-subgroups{type: 'checkbox', ng-model: 'group.membersCanCreateSubgroups'}
-          %span{translate: 'edit_group_form.permissions.members_can_create_subgroups'}
-
-      .checkbox
-        %label{for: 'group-members-can-start-discussions'}
-          %input.edit-group-form__members-can-start-discussions#group-members-can-start-discussions{type: 'checkbox', ng-model: 'group.membersCanStartDiscussions'}
-          %span{translate: 'edit_group_form.permissions.members_can_start_discussions'}
-
-      .checkbox
-        %label{for: 'group-members-can-edit-discussions'}
-          %input.edit-group-form__members-can-edit-discussions#group-members-can-edit-discussions{type: 'checkbox', ng-model: 'group.membersCanEditDiscussions'}
-          %span{translate: 'edit_group_form.permissions.members_can_edit_discussions'}
-
-      .checkbox
-        %label{for: 'group-members-can-edit-comments'}
-          %input.edit-group-form__members-can-edit-comments#group-members-can-edit-comments{type: 'checkbox', ng-model: 'group.membersCanEditComments'}
-          %span{translate: 'edit_group_form.permissions.members_can_edit_comments'}
-
-      .checkbox
-        %label{for: 'group-members-can-raise-motions'}
-          %input.edit-group-form__members-can-raise-motions#group-members-can-raise-motions{type: 'checkbox', ng-model: 'group.membersCanRaiseProposals'}
-          %span{translate: 'edit_group_form.permissions.members_can_raise_motions'}
-
-      .checkbox
-        %label{for: 'group-members-can-vote'}
-          %input.edit-group-form__members-can-vote#group-members-can-vote{type: 'checkbox', ng-model: 'group.membersCanVote'}
-          %span{translate: 'edit_group_form.permissions.members_can_vote'}
+      .edit-group-form__basic
+        .edit-group-form__name
+          %label{for: 'group-name', translate: 'edit_group_form.name'}
+          %input.form-control#group-name{ng-required: true, ng-model: 'group.name', maxlength: '255'}
+          %validation_errors{subject: 'group', field: 'name'}
+
+        .edit-group-form__description
+          %label{for: 'group-description', translate: 'edit_group_form.description'}
+          %textarea.form-control#group-description{msd-elastic: 'true', ng-model: 'group.description'}
+
+        .edit-group-form__privacy-statement
+          {{privacyStatement()}}
+
+        -# Parent Group Privacy Options
+        .edit-group-form__privacy.form-group{ng-if: 'group.isParent()'}
+          %label{for: 'group-privacy', translate: 'edit_group_form.group_privacy'}
+          .edit-group-form__radio-list-item
+            %label
+              %input{type: 'radio', ng-model: 'group.groupPrivacy', value: 'open', name: 'groupPrivacy'}
+              %span{translate: 'edit_group_form.group_privacy_open'}
+          .edit-group-form__radio-list-item
```

Comment:
I'd love to see this be a directive which takes a value and translate key; it'd clarify it a ton for me to see this on say 10 lines, instead of 30.

```
%group_privacy_option{name: 'open', label: 'edit_group_form.privacy_open'}
```
(might have to pass the translate key through with '@' instead of '=' on the directive)